src/components/metrics/card/index.js

```diff
@@ -178,10 +190,26 @@ class MetricCard extends Component {
     return errors ? errors.filter(e => !!e) : []
   }
 
+  showSimulation() {
+    const stats = _.get(this.props, 'metric.simulation.stats')
+    if (stats && _.isFinite(stats.mean) && _.isFinite(stats.stdev) && _.isFinite(stats.length)) {
+      return (stats.stdev === 0 || (stats.length > 5))
```

Comment:
I think you may want 

`return stats.stdev !== 0`

I'm not sure the length check is relevant and I don't know why you'd want to return true if `stdev === 0` as you have now.